src/logger.js

```diff
@@ -0,0 +1,45 @@
+import CoreObject from './metal/core-object';
+
+const Logger = CoreObject.extend({
+  /**
+   * Wrapper around the console log so in the future we can have better dev output.
+   * Also allows us to disable output in production.
+   * @class Logger
+   * @constructor
+   */
+  constructor() {
+  },
+
+  /* eslint-disable no-console */
+  log() {
+    console.log(...this.tagLog(...arguments));
```

Comment:
I like that wrap method. It definitely keeps things DRY, it's even similar to how I tested the methods. As for `console.log` I can take it out. It doesn't really matter and also you are quite correct. We should be precise about logging intent if we are outputting to the console in an environment that isn't our own.